src/main/java/com/nekromant/telegram/controller/PaymentDetailsRestController.java

```diff
@@ -29,12 +31,22 @@ public class PaymentDetailsRestController {
     private ModelMapper modelMapper;
     @Autowired
     private ObjectMapper objectMapper;
+    @Autowired
+    private ResumeAnalysisRequestService resumeAnalysisRequestService;
 
     @PostMapping(value = "/paymentCallback")
```

Comment:
https://restfulapi.net/resource-naming/

```suggestion
    @PostMapping(value = "/payment-callback")
```

Reply:
а это еще до меня было сделано)